go/client/chat_common.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2016 Keybase, Inc. All rights reserved. Use of
```

Comment:
Probably need a better place for this function but I couldn't find any/think of a better name.